src/main/java/eu/arrowhead/application/skeleton/consumer/ConsumerMain.java

```diff
@@ -0,0 +1,397 @@
+package eu.arrowhead.application.skeleton.consumer;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.ApplicationArguments;
+import org.springframework.boot.ApplicationRunner;
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.http.HttpMethod;
+import org.springframework.util.Assert;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import ai.aitia.arrowhead.application.library.ArrowheadService;
+import eu.arrowhead.application.skeleton.consumer.data.AuthRule;
+import eu.arrowhead.application.skeleton.consumer.data.AuthorizationIntraCloudListResponseDTO;
+import eu.arrowhead.application.skeleton.consumer.data.AuthorizationIntraCloudRequestDTO;
+import eu.arrowhead.application.skeleton.consumer.data.AuthorizationIntraCloudResponseDTO;
+import eu.arrowhead.application.skeleton.consumer.data.ServiceDefinitionsListResponseDTO;
+import eu.arrowhead.application.skeleton.consumer.data.ServiceInterfacesListResponseDTO;
+import eu.arrowhead.application.skeleton.consumer.data.SystemListResponseDTO;
+import eu.arrowhead.common.CommonConstants;
+import eu.arrowhead.common.SSLProperties;
+import eu.arrowhead.common.Utilities;
+import eu.arrowhead.common.dto.shared.ServiceDefinitionResponseDTO;
+import eu.arrowhead.common.dto.shared.ServiceInterfaceResponseDTO;
+import eu.arrowhead.common.dto.shared.SystemResponseDTO;
+
+@SpringBootApplication
+@ComponentScan(basePackages = { CommonConstants.BASE_PACKAGE, "ai.aitia" })
+public class ConsumerMain implements ApplicationRunner {
+
+	// =================================================================================================
+	// members
+
+	@Autowired
+	private ArrowheadService arrowheadService;
+
+	@Autowired
+	private SSLProperties sslProperties;
+
+	@Value("#{${default_interface_name_list}}")
```

Comment:
the expression here should be a constant.
Also, I think ${default_interface_name_list} should be enough to load a list from properties